sgkit/stats/aggregation.py

```diff
@@ -0,0 +1,53 @@
+import xarray as xr
+from xarray import DataArray, Dataset
+
+
+def allele_count(ds: Dataset) -> DataArray:
+    """Compute allele count from genotype calls.
+
+    Parameters
+    ----------
+    ds : Dataset
+        Genotype call dataset such as from
+        `sgkit.create_genotype_call_dataset`.
+
+    Returns
+    -------
+    variant/allele_count : DataArray
+        Allele counts with shape (variants, alleles) and values
+        corresponding to the number of non-missing occurrences
+        of each allele.
+
+    Examples
+    --------
+
+    >>> import sgkit as sg
+    >>> from sgkit.testing import simulate_genotype_call_dataset
+    >>> ds = simulate_genotype_call_dataset(n_variant=3, n_sample=2, seed=1)
+    # TODO: Use solution for https://github.com/pystatgen/sgkit/issues/37 here instead
+    >>> ds['call/genotype'].to_series().unstack().astype(str).apply('/'.join, axis=1).unstack()
+    samples 0   1
+    variants
+    0       1/0	1/0
+    1       1/0	1/1
+    2       0/1	1/0
+    3       0/0	0/0
+
+    >>> allele_count(ds)
```

Comment:
```suggestion
    >>> sg.allele_count(ds)
```

Which reminds me, are we running doctests as part of our test suite? If not, would recommend.

Reply:
https://github.com/pystatgen/sgkit/pull/36/commits/802c8d72dd1289ff4901db24755ff44ca51ec715#diff-e2edcf5f78ccf0e17093a6cbe4afeda6R35